CMS/static/scss/subscription.scss

```diff
@@ -30,7 +30,7 @@
   &__label {
     font-weight: initial;
     margin-bottom: 0px;
-    color: #666666;
+    color: #262626;
```

Comment:
I see you have added `$body-text-color` for this colour - use the variable names everywhere you can. same goes for the link colours

changing text colour across the site makes me a touch nervous but using variables would make me feel easier. If we need to tune the values a bit it we'll be able to do it in a couple of keystrokes